taichi/python/export_lang.cpp

```diff
@@ -265,6 +265,95 @@ void export_lang(py::module &m) {
       .def_readwrite("metric_values",
                      &KernelProfileTracedRecord::metric_values);
 
+  // Export ASTBuilder
```

Comment:
In the future we should not put actual `ASTBuilder` logic in `export_lang`. But it's OK for this PR.